### DISCUSSION TITLE
Python / pip packages not available in path

### DISCUSSION THREAD

**Discussion by richstokes** [Homebrew/discussions#3728] [Everyday usage]
Basically running into this issue, https://github.com/opacam/Cohen3/issues/55, but think it might be a pip3 thing rather than specific to this package.

Installed python and pip via home brew on Mac, but I think I'm missing something to get packages installed via pip3 on path.

 

Side note: but why is it so hard to install a sensible python environment? python vs python3, pip vs pip3, paths dont get set correctly, etc. Its a nightmare. IMO brew should install python with working defaults and get one up and running as quickly/easily possible.

**Comment by SMillerDev** [comment]
It's really hard to help without know what you did or what the result was.

**Comment by richstokes** [comment]
Fair enough - maybe I should reverse the question; is there a known-good way to install python/pip via brew? I must have tried about 50 different ways over the years and there's always something broken / doesn't work right

**Reply by SMillerDev** [reply]
`brew install python` has always worked well for me

**Comment by kerim** [comment]
I'm having this same problem. Via a suggestion I found somewhere I found that pip is available here:

ls /opt/homebrew/opt/python@3.9/libexec/bin

But running "pip" still gives errors. Uninstalling and re-installing Python does not help. I think there are ways to tell the system where to look for pip, but I'm not sure how to do this?

**Comment by kerim** [comment]
Interestingly, after I ran "python3.9 -m pip install --upgrade pip" pip is now working.
